inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/job/JobManager.java

```diff
@@ -241,6 +241,8 @@ public Runnable dbStorageCheckThread() {
             while (isRunnable()) {
                 try {
                     jobProfileDb.removeExpireJobs(jobDbCacheTime);
+                    Map<String, String> jobStateMap = jobProfileDb.getJobsState();
```

Comment:
Excuse me, why just print the state into the log, it seems not useful.

Reply:
check local job, submit state to manager in the future.